src/components/posterior_component/posterior.sh

```diff
@@ -100,6 +100,22 @@ run_posterior() {
         source ${GEOSChemEnv}
     fi
 
+    if "$OptimizeBCs"; then
+        if "$KalmanMode"; then
+            period_i=$((period_i+1))
+            inv_result_path="${RunDirs}/kf_inversions/period${period_i}/inversion_result.nc"
+        else
+            inv_result_path="${RunDirs}/inversion/inversion_result.nc"
+        fi
+        # set BC optimal delta values
+        PerturbBCValues=$(generate_optimized_BC_values $inv_result_path)
+        # turn on BC optimization for the corresponding edge and revert emission perturbation
```

Comment:
Minor thing -- this comment says "and revert emission perturbation" but it doesn't look like that happens or is necessary.